### PR TITLE
[Arm32] Fix legacy backend build when FEATURE_READYTORUN_COMPILER is not defined

### PR DESCRIPTION
**Pull request by jashook**

### CODE REVIEW

**Comment by jashook** [comment]
@BruceForstall @jkotas ptal
/cc @dotnet/jit-contrib @sergign60

**Comment by jashook** [comment]
The break originally came in with #12104

File: src/jit/codegenlegacy.cpp [hunk 1]
```diff
@@ -19201,12 +19201,14 @@ regMaskTP CodeGen::genCodeForCall(GenTreeCall* call, bool valUsed)
                 {
                     noway_assert(helperNum != CORINFO_HELP_UNDEF);
 
+                    #ifdef FEATURE_READYTORUN_COMPILER
```
Comment (jkotas) [inline src/jit/codegenlegacy.cpp]: Nit: I think that the convention is to start the ifdefs at the start of the line

File: src/jit/codegenlegacy.cpp [hunk 2]
```diff
@@ -19201,12 +19201,14 @@ regMaskTP CodeGen::genCodeForCall(GenTreeCall* call, bool valUsed)
...
                     if (call->gtEntryPoint.addr != NULL)
                     {
                         accessType = call->gtEntryPoint.accessType;
                         addr       = call->gtEntryPoint.addr;
                     }
                     else
+                    #endif FEATURE_READYTORUN_COMPILER
```
Comment (BruceForstall) [inline src/jit/codegenlegacy.cpp]: This should be:
```
#endif // FEATURE_READYTORUN_COMPILER
```

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT x64 Debug Build and Test

**Pull request merged**